ctms/bin/acoustic.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+"""Schedule contacts to be synced to Acoustic."""
+import os
+import sys
+
+import click
+
+from ctms import config
+from ctms.crud import (
+    bulk_schedule_acoustic_records,
+    create_acoustic_field,
+    create_acoustic_newsletters_mapping,
+    delete_acoustic_field,
+    delete_acoustic_newsletters_mapping,
+    get_all_acoustic_fields,
+    get_all_acoustic_newsletters_mapping,
+    get_contacts_from_newsletter,
+    get_contacts_from_waitlist,
+)
+from ctms.database import SessionLocal
+from ctms.exception_capture import init_sentry
+from ctms.log import configure_logging
+
+
+@click.group()
+@click.pass_context
+def cli(ctx):
+    ctx.obj["dbsession"] = SessionLocal()
+    settings = config.Settings()
+    configure_logging(logging_level=settings.logging_level.name)
+    init_sentry()
+
+
+@cli.command()
+@click.option("--email-list", type=click.File("r"))
+@click.option("--newsletter")
+@click.option("--waitlist")
+@click.pass_context
+def resync(ctx, email_list=None, newsletter=None, waitlist=None):
```

Comment:
Can we add some typing here and/or rename `email_list` to make it obvious that we're passing a file path and not say `--email-list a@example.com b@example.com ...`. I know you have it in the documentation, and `--help` is helpful too:
```
$ ctms/bin/acoustic.py resync --help
Usage: acoustic.py resync [OPTIONS]

  CTMS command to sync contacts with Acoustic.

Options:
  --email-list FILENAME
  ...
```

but "list" is kind of generic, and got me when I was only looking at the function in the context of the test case you wrote.

Also, knowing that `email_list` is a file object and not a `pathlib.Path` would be helpful.